### DISCUSSION TITLE
Allow word: and propagate_case: to be defined globally

### DISCUSSION THREAD

**Discussion by DaWolfi** [espanso/espanso#1242] [Feature requests and Ideas]
Hi there,

thanks for your really great application!

I'm playing around with transforming old autohotkey replacement files, and most of the trigger strings I used in these files would need the "word: true" setting.
Because some files contain many terms, it would make sense for me to define this setting globally, for example at the beginning of the file:

name: autocorrect_de
parent: default
word: true

matches:
….

I think it would be easier in this case to define the few exceptions in the triggers where they are absolutely necessary.
The same then applies analogous to the setting propagate_case:

BR
Wolfgang

**Comment by federico-terzi** [comment]
Hey @DaWolfi,

Thank you for the feedback! This might be useful indeed. I've written a comment about a similar use-case in May: https://github.com/federico-terzi/espanso/issues/632#issuecomment-849786271

I'll add it to the roadmap, thanks!

Cheers :)

**Comment by DaWolfi** [comment]
Thank you for considering these points!

Take care, stay healthy!
